test/app/src/TestApp.ts

```diff
@@ -370,7 +370,7 @@ export default class TestApp {
     const authClient = oktaSign.authClient;
 
     // email verify callback
-    if (authClient.idx.isEmailVerifyCallback(window.location.search)) {
+    if (authClient.idx?.isEmailVerifyCallback(window.location.search)) {
```

Comment:
nit: ? -> !

Reply:
`.idx` doesn't seem to be guaranteed to exist when using `classic` auth-js bundle

Reply:
`.idx` doesn't seem to be guaranteed to exist when using 'classic' auth-js bundle